doc/source/io.rst

```diff
@@ -1258,7 +1258,8 @@ Files with Fixed Width Columns
 
 While ``read_csv`` reads delimited data, the :func:`read_fwf` function works
 with data files that have known and fixed column widths. The function parameters
-to ``read_fwf`` are largely the same as `read_csv` with two extra parameters:
+to ``read_fwf`` are largely the same as `read_csv` with two extra parameters, as
+well as a behavioral change in the ``delimiter`` parameter:
```

Comment:
I would reword as : "and a different usage of the ``delimiter`` parameter" instead of "as well as...parameter"